tools/trap/HowToUseTrap.md

```diff
@@ -49,7 +49,11 @@ Device Drivers -> Block-to-character(BCH) Support
 ```
 
 ## How to parse RAMDUMP
-TRAP Script provides two interfaces: CUI and GUI
+TRAP Script provides two interfaces:  
+1. [CUI](#trap-using-cui)  
+2. [GUI](#trap-using-gui)  
+
+Note: To get Debug Symbols/Crash point using assert logs [CUI](#trap-using-cui), enabling memory dumps [Build Steps](#how-to-enable-memory-dumps) is not mandatory.
```

Comment:
Let's keep the same format for NOTE.
Here  is an example
```
>**Note**
>As the build script is running based on Docker, it requires *sudo* for root permission.
>To run Docker without *sudo*, refer to https://docs.docker.com/install/linux/linux-postinstall/.
```

Reply:
Updated